src/components/timers/Stopwatch.js

```diff
@@ -1,3 +1,48 @@
-const Stopwatch = () => null;
+import Button from "../generic/Button";
+import SetTimes from "../generic/SetTimes";
+import { useRunTimers } from "../../utils/mainHook.js";
+
+const Stopwatch = () => {
+    /**
+    * extract the returned functions and stated values from the custom hook this way
+    * the functions can be called within the onClick prop and the state values can be
+    * called in the clockface div
+    *
+    * also, identify the type of timer and the ids within the timers to
+    * properly set the values in the custom hook
+    * */
+    const {
+        startTimer,
+        pauseTimer,
+        resetTimer,
+        selectedMinute,
+        selectedSecond,
+        // endTimer
+    } = useRunTimers("countup", "sw-min", "sw-sec", "sw-start", "sw-pause", "sw-reset");
```

Comment:
Neat use of custom hooks. When I start to get this many arguments passing into a hook, I like to pass an object in so that I can use the object keys to see what the arguments are. something like

```suggestion
    } = useRunTimers({type: "countup",  ... });
```